dynoid/dynoid.go

```diff
@@ -142,11 +143,28 @@ func LocalTokenPath(audience string) string {
 	return fmt.Sprintf("/etc/heroku/dyno-id/%s/token", audience)
 }
 
+type osReader struct{}
+
+func (*osReader) Open(name string) (fs.File, error) {
+	return os.Open(name)
+}
+
+func (*osReader) ReadFile(name string) ([]byte, error) {
+	return os.ReadFile(name)
+}
+
+// DefaultFS is used by [ReadLocal] and [ReadLocalToken] to retrieve tokens.
+//
+// By default they are retrieved via [os.Open] and [os.ReadFile].
+//
+// This is useful when testing code that uses DynoID.
+var DefaultFS fs.ReadFileFS = &osReader{}
```

Comment:
Would `LocalFS` maybe be better to match the method naming?

Reply:
I was just following the convention from other Go packages in the stdlib. But then again they do have a set of methods that works on the default and another set that allows you to instantiate or pass in your own. :shrug: 

I think I'll leave it for now and we can re-visit as we play with it.